### PR TITLE
fixes #509 - Nav link alignment fixes

### LINKED ISSUE #509: 🕶️ Add more vertical space between the nav links on the Fuchsia site 🕶️
Use up the pink space below `TV`.

Screen Shot 2020-06-23 at 8 20 09 am

### PR DESCRIPTION
**Pull request by shravanshetty1**

### CODE REVIEW

File: docs/_includes/primary-nav-items.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ul>
+<ul= class="nav-links">
```
Comment (jbampton) [inline docs/_includes/primary-nav-items.html:1]:
```suggestion
<ul class="nav-links">
```

**Review by jbampton — CHANGES_REQUESTED** [review]

File: docs/_includes/primary-nav-items.html (hunk 1 above)
Comment (shravanshetty1) [reply docs/_includes/primary-nav-items.html:1]: Any suggestions for code formatter for html css javascript on vscode

**Comment by jbampton** [comment]
You should start with EditorConfig.  Download a plugin for your editor here -> https://editorconfig.org/#download

We already started using it -> https://github.com/slurpcode/slurp/blob/master/.editorconfig

**Comment by jbampton** [comment]
We need to start using Prettier which we can add as a test.

Works on a lot of files.

 https://prettier.io/

**Review by jbampton — APPROVED** [review]
💃

**Pull request merged**